composer/models/huggingface.py

```diff
@@ -96,13 +96,19 @@ def forward(self, batch):
         return output
 
     def loss(self, outputs, batch):
-        return outputs['loss']
+        if self.config.use_return_dict:
+            return outputs['loss']
```

Comment:
Question: Is `outputs` always a list and not a scalar int or float? wondering if there will be a index error or something like that? or do we need a check like `if isinstance(outputs, list)` ?

Reply:
It's a tuple or dict.